### ISSUE TITLE
about "test_image.py"

### ISSUE THREAD

**Issue by 1900lxl** [windrunners/TUFusion#1]
How to realize the output of fusion image of any size in "test_image.py"

**Comment by windrunners** [comment]
We have fixed the code, please refer to the new “test_image.py”.

**Comment by 1900lxl** [comment]
but here comes size mismatch, "x_3 = torch.cat((x_DB, x_2), dim=1)" in net.py.
how to solve this problem
Thx.

**Comment by windrunners** [comment]
Modify line 63 in the file "test_image.py" to "img_fusion = _generate_fusion_image(model, strategy_type, ir_img_resize, vis_img_resize, p_type)" or redownload the "test_image.py" file.

**Comment by 1900lxl** [comment]
Thanks for your reply, it works fine now.
Thx.
